react/features/token-data/User.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Implements a user of conference.
+ */
+export default class User {
```

Comment:
@saghul Seems like we use it only in Ring Overlay so it would be quite small change.